meta-lmp-base/classes/lmp-disable-gplv3.bbclass

```diff
@@ -6,4 +6,5 @@ PACKAGECONFIG:remove:pn-curl = "libidn"
 PACKAGECONFIG:remove:pn-bluez5 = "readline mesh"
 PACKAGECONFIG:append:pn-bluez5 = " libedit "
 PACKAGECONFIG:remove:pn-iproute2 = "elf"
+BAD_RECOMMENDATIONS = "wireguard-tools-wg-quick"
```

Comment:
While this is indeed a way to fix this problem, I would prefer to change the wireguard-tools recipe (via pkgconfig or similiar) in order for us to not really require this package in any image (with and without gplv3).

We don't use this tool since we prefer NM to handle the wireguard connections, so we're safe to drop it entirely.